### PR TITLE
Queues view

### PR DESCRIPTION
**Pull request by onegrx**

### CODE REVIEW

File: app/controllers/HomeController.scala [hunk 1]
```diff
@@ -61,6 +63,15 @@ class HomeController @Inject()(db: Database, cc: ControllerComponents, service:
     }
   }
 
+  def getMockQueue(patient: Patient): Future[Seq[(ClinicQueue, Option[Ticket], Doctor)]] = {
+    val queue = PublicQueue(1, DateTime.now().minusHours(1), DateTime.now(), Seq(1, 2, 3))
+    val ticket = Some(Ticket(1, new Patient(7, "John", "Doe")))
+    val doctor = new Doctor(6, "Doctor", "House")
+    Future {
```
Comment (bartpr) [inline app/controllers/HomeController.scala]: You can just Future(something) for one instruction expression

File: app/controllers/HomeController.scala [hunk 2]
```diff
@@ -84,6 +95,20 @@ class HomeController @Inject()(db: Database, cc: ControllerComponents, service:
     service.close(queueId)
   }
 
+  def queues = Action.async {
+    val patient = new Patient(2, "Foo", "Bar")
+    val result = for {
```
Comment (bartpr) [inline app/controllers/HomeController.scala]: somethink like queues.map(q =>Ok(views.html.queues(q)) should have the same result and it's more readable

File: app/controllers/HomeController.scala [hunk 3]
```diff
@@ -84,6 +95,20 @@ class HomeController @Inject()(db: Database, cc: ControllerComponents, service:
...
+      queues <- getMockQueue(patient)
+    } yield {
+      Ok(views.html.queues(queues))
+    }
+
+    result.recover {
```
Comment (bartpr) [inline app/controllers/HomeController.scala]: Is it necessary?

Comment (onegrx) [reply app/controllers/HomeController.scala]: When using `Action.async` it seems to be necessary

File: app/controllers/HomeController.scala (hunk 1 above)
Comment (onegrx) [reply app/controllers/HomeController.scala]: 👍

File: app/controllers/HomeController.scala (hunk 2 above)
Comment (onegrx) [reply app/controllers/HomeController.scala]: You're right

**Pull request merged**